### DISCUSSION TITLE
v0.7.1

### DISCUSSION THREAD

**Discussion by Hayao-H** [Hayao-H/Niconicome#360] [一般]
# 概要
アプリケーションを実行出来ない問題を修正しました。
大変申し訳ありませんでした。 :bow:
## 修正
- ```chrome```ディレクトリーが存在しない場合に実行出来ない問題を修正 #353

<hr /><em>This discussion was created from the release <a href='https://github.com/Hayao-H/Niconicome/releases/tag/v0.7.1'>v0.7.1</a>.</em>

**Comment by amkb018** [comment]
可能でしたら、画面の表示サイズを小さめに表示できるようにして欲しいです。

ノートPCのディスプレイ解像度が1366x768なので、最大化しても全てが表示
されないので、設定やDLするときに少し面倒です。

よろしくお願いします。

**Reply by Hayao-H** [reply]
ご要望ありがとうございます。
現在少しずつ表示関連をカスタマイズできるようにしておりますので、もうしばらくお待ち下さい。
よろしくおねがいします。
